### ISSUE TITLE
Need better instructions to install ZeroBlog. Currently ZeroBlog doesn't install from git.

### ISSUE THREAD

**Issue by yurivict** [HelloZeroNet/ZeroBlog#24]
Your instructions ask to clone the 1BLog... site. But users should be able to install from git too, because that's how software is usually installed. There are also problems in instructions like there is no file default.json-default that instructions mention.

I am trying to write a script to clone from git, but it fails for unclear reason. zeronet siteSign skips data/users/content.json for some reason.

```
cd data/{your-new-site}
git clone https://github.com/HelloZeroNet/ZeroBlog.git &&
mv ZeroBlog/* . &&
rm -rf ZeroBlog &&
mv data-default data &&
mv data/users/content-default.json data/users/content.json
perl -pi -e "s/1BLogC9LN4oPDcruNz3qo1ysa133E9AGg8/`pwd | sed -e 's/.*\///g'`/g" content.json &&
perl -pi -e "s/\"domain.*//g" content.json &&
perl -pi -e "s/\"files\": \{\},/\"files\": {}, \"inner_path\": \"data\/users\/content.json\",/g" data/users/content.json
```

I think you need to make instructions to create a new blog easier, and they should start from "git clone". It is best to have a script cloning the github repository. You probably also want to remove "domain": "Blog.ZeroNetwork.bit" from contents.json because only you own it, and most people probably will not register a domain.

**Comment by HelloZeroNet** [comment]
you have to sign the other content.json file using `zeronet.py siteSign address --inner_path data/users/content.json` after that it should work.

but if you use the built-in clone feature it will do that for you and its also removing the domain entry

**Comment by yurivict** [comment]
Yes, `zeronet.py siteSign {address} --inner_path data/users/content.json` was missing.

So it would be great if ZeroBlog page had accurate instructions including that one command. I think siteSign should also replace "address" in content.json, and instructions should look like this:

```
cd data/{your-new-site}
git clone https://github.com/HelloZeroNet/ZeroBlog.git &&
mv ZeroBlog/* . &&
rm -rf ZeroBlog &&
mv data-default data &&
mv data/users/content-default.json data/users/content.json &&
ADDR=`pwd | sed -e 's/.*\///g'` &&
zeronet siteSign $ADDR &&
zeronet siteSign $ADDR --inner_path data/users/content.json
```
